src/contract-interact.ts

```diff
@@ -282,9 +254,9 @@ function createDummyTx(
       winston: tx.quantity,
     },
     block: {
-      height,
-      id: current,
-      timestamp: parseInt((Date.now() / 1000).toString(), 10),
```

Comment:
main change in this PR - instead of setting block timestamp of the dummyTx to `Date.now()` (which was inconsistent with the rest of the block data) - timestamp is now taken from current's block timestamp.